apps/bb2_tools/admin.py

```diff
@@ -663,25 +663,37 @@ class V2UsersViewerAdmin(ReadOnlyAdmin):
     list_filter = (UserTypeFilter, V2Filter)
 
     def get_type(self, obj):
-        return obj.userprofile.user_type
+        print("====================V2UsersViewerAdmin.get_type()==============================")
```

Comment:
@JFU-GIT The `print` debugging statements should be removed in this file: apps/bb2_tools/admin.py